pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -66,6 +66,8 @@ const (
 	ZeroedThickDiskType       = "zeroedThick"
 	VolDir                    = "kubevols"
 	RoundTripperDefaultCount  = 3
+	DummyVMName               = "kubernetes-helper-vm"
```

Comment:
Ok. I guess since the VM is not deleted this is true only for the first and second volume creation race.